Add french language

kindly opened the pull request:
<!---
@huboard:{"order":324.0,"milestone_order":324,"custom_state":""}
-->

kindly committed:
Add french language

timgdavies commented:
We still need to review the French Translation before this is fully merged in.

Bjwebb committed:
Merge branch '1.0-dev' into 1.0-dev-add-french

Bjwebb committed:
Update french translation from transifex

Bjwebb committed:
Add missing french governance.po

Not sure why this wasn't pulled from transifex before. Was able to fetch
it with:
tx pull -r open-contracting-standard-1-0.support--governance -l fr

Bjwebb committed:
Pull updated french translations from transifex

Bjwebb committed:
Update sphinx and recommonmark dependencies

Bjwebb committed:
Seperate text from html tags to ensure it's picked up for translation

timgdavies committed:
Updating french translations for testing

Bjwebb committed:
Force pull french translations from transifex

tx pull -l fr -f

Bjwebb committed:
Revert "Update sphinx and recommonmark dependencies"

This reverts commit d6ea5ee10c9b480533b44d323938a624062a9904.

Bjwebb committed:
Fix some formatting issues in the french translation

Bjwebb committed:
Force pull french translations from transifex

tx pull -l fr -f

Bjwebb commented:
French translation has now been reviewed.
